lib/validation/validation_form.dart

```diff
@@ -0,0 +1,54 @@
+bool validateEmailAddress(String input) {
+  const emailRegex =
+      r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$';
+
+  if (RegExp(emailRegex).hasMatch(input)) {
+    return true;
+  } else
+    return false;
+}
+
+String? passwordValidator(String? value) {
+  if (value == null || value.isEmpty) {
+    return "Password is required";
```

Comment:
Add localisation

Reply:
To be able to use context in this file, you have to create a navigator key instance. Something like:
 class NavigationService {
  static final navigatorKey = GlobalKey<NavigatorState>() = NavigationService.navigatorKey;
}
Then you can use it like navigatorKey.currentContext